lib/Service/LoginService.php

```diff
@@ -325,9 +325,26 @@ public function login($profile)
             }
         }
 
+        $this->completeLogin($user, $userPassword, $userSession, $request);
+
         return [$user, $userPassword];
     }
 
+    public function completeLogin($user, $userPassword, $userSession, $request)
+    {
+        $userSession->getSession()->regenerateId();
+        $tokenProvider = \OC::$server->query(DefaultTokenProvider::class);
```

Comment:
You are missing the import to `DefaultTokenProvider` here. Currently, I can't log in.